soc/wch/ch32v/qingke_v4b/Kconfig

```diff
@@ -8,3 +8,6 @@ config SOC_SERIES_QINGKE_V4B
 	select RISCV_ISA_EXT_C
 	select RISCV_ISA_EXT_ZICSR
 	select RISCV_ISA_EXT_ZIFENCEI
+	select RISCV_ALWAYS_SWITCH_THROUGH_ECALL
```

Comment:
This is needed on the v2a, but we didn't see the same lockup on the v2c or v4f. Could you confirm that this is needed?

Reply:
I've verified on v4f (CH32V305), but don't have v2c on hand to verify. Would appreciate help testing, otherwise I can remove this from there for now.